Sources/SwiftProcessing/Core/UIKitElements/Capture.swift

```diff
@@ -0,0 +1,310 @@
+import UIKit
+import AVFoundation
+import Vision
```

Comment:
is this import needed?

Reply:
Yes is required for line 12, sequence handler